<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QDialog" name="Console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="database_tab">
      <attribute name="title">
       <string>Database</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextBrowser" name="database_output"/>
       </item>
       <item>
        <widget class="QGroupBox" name="database_command_group">
         <property name="title">
          <string>Command</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="database_query"/>
          </item>
          <item>
           <widget class="QPushButton" name="database_run">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
